src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -131,6 +131,23 @@ public Color BorderColor
 			set => SetValue(BorderColorProperty, value);
 		}
 
+		/// <summary>
+		/// Backing BindableProperty for the <see cref="CornerRadius"/> property.
+		/// </summary>
+		public static readonly BindableProperty CornerRadiusProperty =
```

Comment:
Could we create a constant like https://github.com/xamarin/Xamarin.Forms/blob/caab66bcf9614aca0c0805d560a34e176d196e17/Xamarin.Forms.Core/BorderElement.cs#L7?, why different values by platform?

Reply:
It was initialized like this before my change.

https://github.com/xamarin/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs#L277

Setting those default values will prevent to modify the badgeview appearance when people will update the nuget package.

I will add a sample OK, thanks for your feedback